OneCommunityGlobal/HighestGoodNetworkApp
# Mohan - Made Changes For add or delete teams without saving

# Description
![image](https://github.com/user-attachments/assets/d4030f52-476a-4043-9cb3-a51419860551)

## Related PRS (if any):
This frontend PR is related to the previous PR https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2607



## How to test:
1. check into current branch
2. do npm install and npm run start:local to run this PR locally
3. Clear site data/cache
4. log as admin user
5. Open a User Profile-> Teams->Assign Teams
6. Now Perform Delete Teams The Functionality Should Work without saving the changes.
7. Verify If the Changes are working for both the Desktop and Mobile Version

## Screenshots or videos of changes:

https://github.com/user-attachments/assets/3baca2c7-e810-4350-b7d0-dc08c3f6eea6
